python_modules/libraries/dagster-azure/dagster_azure_tests/adls2_tests/test_adls2_file_manager.py

```diff
@@ -169,6 +173,7 @@ def test_op(context):
         MockADLS2Resource.assert_called_once_with(
             storage_account=resource_config["storage_account"],
             credential=ADLS2Key(key=resource_config["credential"]["key"]),
+            cloud_type=resource_config.get("cloud_type", "public"),
```

Comment:
[3] Is there a case where it won't be `public`? Or a case where `cloud_type` won't be in `resource_config`? It looks like `cloud_type` was not added to `resource_config` in the test, so I'm not sure why we use `get()` on `resource_config`.

Reply:
Good call. `cloud_type` is an optional config parameter that defaults to `public`, so changed this to just be `public` since it's not in the `resource_config` for this test.